docs/Makefile

```
@@ -0,0 +1,11 @@
+SPHINXOPTS    ?= -W --keep-going
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)